src/assets/icons/rocking_chair_blue.svg

```diff
@@ -0,0 +1 @@
+<svg id="blue_svg_2px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 96"><defs><style>.cls-1{fill:#00008f;}</style></defs><title>deckchair</title><path class="cls-1" d="M81.94,76.88a3.71,3.71,0,0,0-4.41-.7,42.09,42.09,0,0,1-6.48,2.66A62.64,62.64,0,0,1,67,61.8h2.71a5.11,5.11,0,0,0,0-10.22h-3.1V36h3.69a1,1,0,0,0,0-2l-34.9,0C33.33,23.87,29.68,14.8,24.71,7.8a4.25,4.25,0,0,0-2.83-1.75,4.38,4.38,0,0,0-4.3,6.81c6.68,9.44,10.66,24.52,10.66,40.34a71.26,71.26,0,0,1-4.36,25.28,40.78,40.78,0,0,1-5.44-2.3A3.74,3.74,0,0,0,15,82.82C23.64,87.45,35.36,90,48,90s24.37-2.56,33-7.2a3.75,3.75,0,0,0,.92-5.92ZM72.79,56.69a3.11,3.11,0,0,1-3.1,3.11H30c.15-2,.23-4.07.24-6.22h14a.57.57,0,0,0,.45,0H54.81a.57.57,0,0,0,.45,0H65.36a.57.57,0,0,0,.45,0h3.88A3.11,3.11,0,0,1,72.79,56.69ZM30.22,51.58A93.92,93.92,0,0,0,28.69,36h5a95.23,95.23,0,0,1,1.53,15.61Zm15.27,0V36H54V51.58Zm19.1,0H56V36h8.55ZM43.48,36V51.58H37.25A97.29,97.29,0,0,0,35.76,36ZM19.75,8.48A2.43,2.43,0,0,1,21.57,8,2.26,2.26,0,0,1,23.08,9c4.75,6.69,8.25,15.35,10.26,25H28.29c-1.84-8.7-4.93-16.39-9.07-22.25A2.37,2.37,0,0,1,19.75,8.48Zm10,53.32H65a64.1,64.1,0,0,0,4.17,17.61A79.06,79.06,0,0,1,48,82.13a77.87,77.87,0,0,1-22.19-3A73.74,73.74,0,0,0,29.8,61.8ZM81,79.77a1.74,1.74,0,0,1-.9,1.27c-8.26,4.42-20,7-32.09,7s-23.8-2.53-32.06-6.94a1.75,1.75,0,0,1-.41-2.79h0a1.72,1.72,0,0,1,1.22-.5,1.71,1.71,0,0,1,.79.19,43.83,43.83,0,0,0,6.64,2.73h0A78.73,78.73,0,0,0,48,84.13c11.84,0,22.94-2.25,30.47-6.17A1.74,1.74,0,0,1,81,79.77Z"/></svg>
```

Comment:
Please use `<path fill="currentColor"`. instead of inline style so that a user can override the color